packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -247,6 +249,43 @@ export const initProvider = () => (dispatch: Dispatch) => {
     return decision.promise;
 };
 
+const getMetadataFilesList = () => async (dispatch: Dispatch) => {
+    const provider = dispatch(getProvider());
+    if (!provider) {
+        return;
+    }
+
+    const listResponse = await provider.getMetadataFilesList();
+    if (!listResponse.success) {
+        return;
+    }
+
+    // remove the extention
+    const trimmedNames = listResponse.payload?.map(name => name.slice(0, -5));
```

Comment:
nit: personally I'd do 
```
name.replace('.mtdt', '');
```
name.slice is cool but one needs to know how it works and also it is not visibile at the first glance why I am doing this operation

Reply:
done